packages/subcommands/src/listeners/PluginChatInputSubcommandNoMatch.ts

```diff
@@ -0,0 +1,12 @@
+import { Listener, type ChatInputCommand } from '@sapphire/framework';
+import { SubcommandPluginEvents, type ChatInputSubcommandNoMatchContext } from '../lib/types/Events';
+
+export class PluginListener extends Listener<typeof SubcommandPluginEvents.ChatInputSubcommandNoMatch> {
+	public constructor(context: Listener.LoaderContext) {
+		super(context, { event: SubcommandPluginEvents.ChatInputSubcommandNoMatch });
+	}
+
+	public override run(_interaction: ChatInputCommand.Interaction, context: ChatInputSubcommandNoMatchContext) {
+		this.container.logger.error(context.message);
```

Comment:
Are these loaded by default? Because if so, they shouldn't be

Reply:
```
		if (options.loadSubcommandErrorListeners !== false) {
			loadListeners();
		}
```
has been in `/src/register.ts` for 2 years